components/storage/blockdevice/COMPONENT_SPIF/SPIFBlockDevice.cpp

```diff
@@ -210,6 +210,7 @@ int SPIFBlockDevice::init()
     // Dummy And Mode Cycles Back default 0
     _dummy_and_mode_cycles = _write_dummy_and_mode_cycles;
     _is_initialized = true;
+    tr_info("Device size: %llu Kbytes", _device_size_bytes / 1024);
```

Comment:
Debug perhaps..
This is normal operation, and should not cause any output by default, because it succeed.